open-ce/utils.py

```diff
@@ -227,6 +227,12 @@ def is_subdir(child_path, parent_path):
     relative = os.path.relpath(child, start=parent)
     return not relative.startswith(os.pardir)
 
+def is_url(to_check):
+    '''
+    Determines if a string is a URL
+    '''
+    return to_check.startswith("http:") or to_check.startswith("https:")
```

Comment:
ssh support?

Reply:
I think this would entail alot more work. If a user wants to use environment files that are in a repo behind a login, they should clone the repo and then use them locally.